release/setup.py

```diff
@@ -12,105 +12,106 @@
-University of Waterloo and the Quantum AI team at Google along with help from
-many other contributors within Google.
+University of Waterloo and the Quantum AI team at Google along with help
+from many other contributors within Google.
 """
+
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
 
 import sys
-
 from datetime import date
+
 from setuptools import Extension
 from setuptools import find_packages
 from setuptools import setup
-from setuptools.dist import Distribution
 from setuptools.command.install import install
+from setuptools.dist import Distribution
 
-
-DOCLINES = __doc__.split('\n')
+DOCLINES = __doc__.split("\n")
 
 
 class InstallPlatlib(install):
-    """Workaround so .so files in generated wheels
-    can be seen by auditwheel."""
+    """Workaround so .so files in generated wheels are visible to auditwheel."""
 
     def finalize_options(self):
         install.finalize_options(self)
         if self.distribution.has_ext_modules():
             self.install_lib = self.install_platlib
 
 
-REQUIRED_PACKAGES = ['cirq-core==1.3.0', 'cirq-google==1.3.0', 'sympy == 1.12']
+REQUIRED_PACKAGES = [
+    "cirq-core==1.3.0",
+    "cirq-google==1.3.0",
+    "sympy==1.14",
+]
+
+# Placed as extras to avoid overwriting existing nightly TF installs.
+EXTRA_PACKAGES = ["tensorflow>=2.16,<2.17"]
 
-# placed as extra to not have required overwrite existing nightly installs if
-# they exist.
-EXTRA_PACKAGES = ['tensorflow == 2.15.0']
-CUR_VERSION = '0.7.4'
+CUR_VERSION = "0.7.5"
 
 
 class BinaryDistribution(Distribution):
-    """This class is needed in order to create OS specific wheels."""
+    """Create OS-specific wheels."""
 
     def has_ext_modules(self):
         return True
 
 
-nightly = False
-if '--nightly' in sys.argv:
-    nightly = True
-    sys.argv.remove('--nightly')
+NIGHTLY_FLAG = False
+if "--nightly" in sys.argv:
+    NIGHTLY_FLAG = True
+    sys.argv.remove("--nightly")
 
-project_name = 'tensorflow-quantum'
-build_version = CUR_VERSION
-if nightly:
-    project_name = 'tfq-nightly'
-    build_version = CUR_VERSION + '.dev' + str(date.today()).replace('-', '')
+PROJECT_NAME = "tensorflow-quantum"
+BUILD_VERSION = CUR_VERSION
+if NIGHTLY_FLAG:
+    PROJECT_NAME = "tfq-nightly"
+    BUILD_VERSION = CUR_VERSION + ".dev" + str(date.today()).replace("-", "")
 
 setup(
-    name=project_name,
-    version=build_version,
-    description=
-    'TensorFlow Quantum is a library for hybrid quantum-classical machine learning.',
-    long_description='\n'.join(DOCLINES[2:]),
-    author='Google Inc.',
-    author_email='no-reply@google.com',
-    url='https://github.com/tensorflow/quantum/',
+    name=PROJECT_NAME,
+    version=BUILD_VERSION,
+    description="Library for hybrid quantum-classical machine learning.",
+    long_description="\n".join(DOCLINES[2:]),
+    author="Google Inc.",
+    author_email="no-reply@google.com",
```

Comment:
Since this file is getting changes for other reasons, it may as well also include an update to these values. The general approach we've taken in other major QAI projects is to use the following pattern, so I think it's worth including in this PR so that another PR isn't needed just for this.

```suggestion
    author="The TensorFlow Quantum Authors",
    author_email="tensorflow-quantum-team@google.com",
```